staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3861,13 +3873,77 @@ func describeNode(node *corev1.Node, nodeNonTerminatedPodsList *corev1.PodList,
 		} else {
 			w.Write(LEVEL_0, "Pods:\tnot authorized\n")
 		}
+
 		if events != nil {
 			DescribeEvents(events, w)
 		}
 		return nil
 	})
 }
 
+// Add helper function to describe ResourceSlices
+func describeNodeResourceSlices(w PrefixWriter, node *corev1.Node, resourceSlices []resourcev1beta1.ResourceSlice) {
+	w.Write(LEVEL_0, "ResourceSlices:\n")
+	w.Write(LEVEL_0, "  Name\t\tDriver\t\tPool\t\tAssociation\n")
+	w.Write(LEVEL_0, "  ----\t\t------\t\t----\t\t-----------\n")
+
+	// Sort slices by name for consistent output
+	sort.Slice(resourceSlices, func(i, j int) bool {
+		return resourceSlices[i].Name < resourceSlices[j].Name
+	})
+
+	for _, slice := range resourceSlices {
+		association := ""
+		// Determine association type
+		switch {
+		case slice.Spec.NodeName != "" && slice.Spec.NodeName == node.Name:
+			association = "Direct node match"
```

Comment:
I think these are the ONLY ones we should show. We should filter this out earlier and not show any others. There can really be a lot of resources slices "available from" a node - think thousands. We shouldn't show the "could be used on this node" as the same as "can ONLY be used on this node". The latter are all we want in a per-node view.

To see the other things, users should look at the resource slices directly.